migrations/0000_createDbs.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS "Users"(
+	id bigserial PRIMARY KEY,
+    "displayName" text NOT NULL,
+    "emailAddress" text UNIQUE NOT NULL,
+    password bytea NOT NULL,
+    "creationTime" timestamp NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "Publications"(
+	id bigserial PRIMARY KEY,
+    "authorId" bigint references "Users"(id) NOT NULL,
+    "sequenceNumber" serial NOT NULL,
```

Comment:
I was thinking about this again. I don't think we need this field. The query is pretty straightforward without it:
```
select * from publications
    where author_id = _user_id
    order by id asc
    limit _number_of_publications_to_show
```